code/main/common/string.brs

```diff
@@ -42,3 +50,22 @@ function _adb_isStringInArray(str as string, array as object) as boolean
 
     return false
 end function
+
+function _adb_stringEqualsIgnoreCase(s1 as dynamic, s2 as dynamic) as boolean
```

Comment:
Good idea!
Should we trim both strings before making the comparison? Or adding this logic in a new function?
https://developer.roku.com/docs/references/brightscript/interfaces/ifstringops.md#trim-as-string

Reply:
Yes, let's add that.